poetry.lock

```diff
@@ -1676,22 +961,809 @@ test = ["covdefaults (>=2.3)", "coverage (>=7.2.7)", "coverage-enable-subprocess
 name = "zipp"
 version = "3.18.1"
 description = "Backport of pathlib-compatible object wrapper for zip files"
+category = "main"
 optional = true
 python-versions = ">=3.8"
-files = [
-    {file = "zipp-3.18.1-py3-none-any.whl", hash = "sha256:206f5a15f2af3dbaee80769fb7dc6f249695e940acca08dfb2a4769fe61e538b"},
-    {file = "zipp-3.18.1.tar.gz", hash = "sha256:2884ed22e7d8961de1c9a05142eb69a247f120291bc0206a00a7642f09b5b715"},
-]
 
 [package.extras]
 docs = ["furo", "jaraco.packaging (>=9.3)", "jaraco.tidelift (>=1.4)", "rst.linker (>=1.9)", "sphinx (>=3.5)", "sphinx-lint"]
 testing = ["big-O", "jaraco.functools", "jaraco.itertools", "more-itertools", "pytest (>=6)", "pytest-checkdocs (>=2.4)", "pytest-cov", "pytest-enabler (>=2.2)", "pytest-ignore-flaky", "pytest-mypy", "pytest-ruff (>=0.2.1)"]
 
 [extras]
-docs = ["Sphinx", "sphinx_rtd_theme"]
+docs = ["Sphinx", "sphinx_rtd_theme", "toml"]
 grpc = ["grpcio", "protobuf"]
 
 [metadata]
-lock-version = "2.0"
+lock-version = "1.1"
```

Comment:
I thought we finally upgraded Poetry?

Reply:
Woops, order of operations... deleted my comment

1.2.2, which is what I have, is the current version per [wiki](https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/70875/Optional-tools-for-uncommon-cases#) and [buildpy](https://dev.azure.com/ni/DevCentral/_git/Tools?version=GCe6a0961a8db1263a7869ff78c147e1032aa39e3e&path=/build/buildpy_poetry/buildpy_poetry/_poetry.py).